packages/teleport/src/services/recordings/recordings.ts

```diff
@@ -12,11 +14,14 @@ export default class RecordingsService {
     const start = params.from.toISOString();
     const end = params.to.toISOString();
 
-    const url = cfg.getClusterEventsRecordingsUrl(clusterId, {
+    const url = cfg.getClusterEventsUrl(clusterId, {
```

Comment:
lets remove this service and move `fetchRecordings` and `makeRecording` methods to the `audit` service.

Reply:
Actually, lets keep it as is, seems cleaner to keep these abstractions separated.

Reply:
actually, lets keep it. Seems cleaner to separate recordings from events this way.